doc/examples/segmentation/plot_rag_boundary.py

```diff
@@ -24,8 +24,10 @@
 edges_rgb = color.gray2rgb(edges)
 
 g = graph.rag_boundary(labels, edges)
+
+fig, ax = plt.subplots()
 lc = graph.show_rag(labels, g, edges_rgb, img_cmap=None, edge_cmap='viridis',
-                    edge_width=1.2)
+                    edge_width=1.2, ax=ax)
 
-plt.colorbar(lc, fraction=0.03)
+fig.colorbar(lc, fraction=0.03)
 io.show()
```

Comment:
```suggestion
plt.show()
```
Isn't it confusing here?